Long prefix build of version 0.29.1

jjhelmus opened the pull request.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jjhelmus commented:
Please ignore until #7 is merged.  Just want to see if this passes the CI

jjhelmus commented:
Woo hoo! :fireworks: 
```
No packages found with binary prefixes shorter than 255 characters.
```

jjhelmus commented:
Rebased this PR against `master` and removed this PR.  If this passes CI this should be good to go!  @ocefpaf and @jakirkham thanks for the timely reviews, merges and encouragement.
